src/trades/__tests__/tradeListSelectors-test.js

```diff
@@ -10,6 +10,7 @@ describe('tradeListSelectors', () => {
         trades: fromJS([]),
         tradingOptions: fromJS([]),
         tradingTimes: fromJS([]),
+        appState: fromJS({ authorized: true }),
```

Comment:
This is indicative of a potential problem I noticed.
A selector depending on too many reducers might not be ideal.
Need discussion between 3 of us.